### ISSUE TITLE
error install photoprism rpi 4 4gb

### ISSUE THREAD

**Issue by adrianoleal** [photoprism/photoprism#547]
Hello everyone.
I have little general knowledge in the area (docker, docker-compose, etc.), for this reason, I apologize for any lack of knowledge of mine, I only know the basics to run what I want and need, always using the most advanced users.
I am delighted with the Photoprism project, I discovered it just over 4 days ago.
The idea is great, congratulations.
I'm looking to install it on rpi 4 4gb, which is running OVM (openmediavault) docker, docker-compose and portainer.
I have some containers running well.
When I try to install photoprism, through docker-compose, I have this output:

root @ server: / home / pi # docker-compose up
Creating network "pi_default" with the default driver
Creating pi_watchtower_1 ... done
Creating pi_photoprism_1 ... done
Attaching to pi_watchtower_1, pi_photoprism_1
photoprism_1 | standard_init_linux.go: 211: exec user process caused "exec format error"
watchtower_1 | time = "2020-10-13T16: 51: 44Z" level = info msg = "Starting Watchtower and scheduling first run: 2020-10-13 16:56:44 +0000 UTC m = + 300.086689978"
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 0
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1
pi_photoprism_1 exited with code 1

It is the exit table that I have when I try to install something that does not run on the rpi platform (arm64).

[file docker-compose.yml](https://github.com/photoprism/photoprism/files/5372944/docker-compose.txt)

**Comment by lastzero** [comment]
Thanks for reporting this, I suspect the error is related to our new entry file.

You should try either using Docker to set the user and group like `user: "1000:1000"` or set GID / UID via environment variable. Maybe it doesn't work because you did both?

**Comment by lastzero** [comment]
Just improved our `entrypoint.sh` script that runs before Docker starts PhotoPrism.

UID/GID handling and the exec call have been changed, so it might work. Can you pull our latest image (available in about one hour if the [build](https://drone.photoprism.app/photoprism/photoprism/159/1/1) doesn't fail) and then test again?

Thank you :)

**Comment by adrianoleal** [comment]
Good night.
I must thank you.
I will try the new image as soon as possible, I believe tomorrow morning. I'm looking forward to it, because I loved the idea of Photoprism.
I intend to be peculiar within my limitations. Be it with some amount of money (lately the exchange rate is terrifying for my country's currency, about 6 to 1, lol) but I want to contribute and encourage the development of this project.
I wish you much success and luck.

**Comment by adrianoleal** [comment]
> Had to restart the build, but now it seems to work. New image for ARM64 is already available.

First of all, thank you for the effort and everything.
I checked everything and made sure everything was correct.
I copied the ".yml" file without making any changes.
After the "docher-compose up" command, the installation went smoothly.
I noticed that Photoprism works.
Tomorrow I will do a wider, more effective scan with my settings.
This problem has been successfully resolved.
Once again, thank you.
Success.
